src/System.Runtime.Extensions/ref/System.Runtime.Extensions.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Runtime.Extensions.csproj" />
+    <Project Include="System.Runtime.Extensions.csproj">
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
No: we have similar precedent on desktop where more surface area is visible via facades.